src/Microsoft.AspNet.Server.Kestrel/Infrastructure/Disposable.cs

```diff
@@ -41,6 +39,5 @@ public void Dispose()
             Dispose(true);
             GC.SuppressFinalize(this);
```

Comment:
Doesn't this patter only matter if this class (or one derived from it) has a finalizer?